sdk/core/azure-core/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 ### Features Added
 
 - When a `RequestFailedException` exception is thrown, the `what()` method now includes information about the HTTP request which failed.
-- Adding option `WinHttpTransportOptions.IgnoreUnknownServerCert`. It can be used to disable verifying server certificate for the `WinHttpTransport`.
+- Adding option `WinHttpTransportOptions.IgnoreUnknownServerCertificate`. It can be used to disable verifying server certificate for the `WinHttpTransport`.
```

Comment:
```suggestion
- Adding option `WinHttpTransportOptions.IgnoreUnknownCertificateAuthority`. It can be used to disable verifying server certificate for the `WinHttpTransport`.
```